src/features/common/Layout/Header/Header.module.scss

```diff
@@ -4,3 +4,64 @@
   primaryColor: $primaryColor;
   primaryFontColor: $primaryFontColor;
 }
+
+.headerBG {
+  height: 250px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background: linear-gradient(
+      to top,
+      rgba(0, 0, 0, 0.5),
+      rgba(0, 0, 0, 0.2),
+      rgba(0, 0, 0, 0),
+      rgba(0, 0, 0, 0)
+    ),
+    url('https://cdn.plant-for-the-planet.org/media/images/app/bg_layer.jpg')
```

Comment:
Is is possible to also try to take the value from our environment variable in an .scss file as in an .tsx file? Or can we rewrite this to load the background image in the related component using this style like we did it for the user profile at https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/f358ad2e551ba81584c090d0ca88c105c59f0b09/src/features/user/UserProfile/screens/IndividualProfile.tsx#L61-L63

Reply:
I guess individual profile uses different header. The header component just has back button and settings button which is going to be used in different pages which does not require other features which individual profile has.

Reply:
close in favor of https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1022#discussion_r627960285